src/datatypes/src/value.rs

```diff
@@ -286,6 +289,26 @@ fn timestamp_to_scalar_value(unit: TimeUnit, val: Option<i64>) -> ScalarValue {
     }
 }
 
+/// Convert [ScalarValue] to [Timestamp].
+/// Return `None` if
+pub fn scalar_value_to_timestamp(scalar: &ScalarValue) -> Option<Timestamp> {
+    match scalar {
+        ScalarValue::Int64(val) => val.map(Timestamp::new_millisecond),
+        ScalarValue::Utf8(Some(s)) => match Timestamp::from_str(s) {
+            Ok(t) => Some(t),
+            Err(e) => {
+                logging::error!("Failed to convert string literal {s} to timestamp, error: {e:?}");
```

Comment:
The error returned by `Timestamp::from_str` is `FromStr`'s associated type `Error` which does not necessarily implemented `ErrorExt`, so it cannot be used in `error!(e; "xxx")`

Reply:
Looks like we could implement ErrorExt for this error. The Error is defined in common_time crate. Otherwise, we should use display instead of debug trait to print the error.